index.html

```diff
@@ -255,6 +255,19 @@ <h3>legend</h3>
           }
         }
 
+        function open_row(row, tr) {
+          var extra_data = format(row.data());
+          if (extra_data !== "") {
+            row.child( format(row.data()) ).show();
+            tr.classList.add('shown');
+          }
+        }
+
+        function close_row(row, tr) {
+          row.child.hide();
+          tr.classList.remove('shown');
```

Comment:
As far as I can tell, the 'shown' class isn't really doing anything.

Reply:
It's used by the CSS to style the row after the target when the item is expanded (i.e., give it the yellow background).  (And that is still "needed" since it's possible to close an item while it remains the target.)